code/_helpers/client.dm

```diff
@@ -48,6 +48,8 @@
 		view_radius = round(max(view_size[1], view_size[2])/2)
 		view = new_size
 
+	SEND_SIGNAL(src, COMSIG_VIEW_SET, new_size)
```

Comment:
should specify this is for clients, `COMSIG_CLIENT_VIEW_SET`

Reply:
You should probably rename it on Daedalus as well